### DISCUSSION TITLE
multi db multitenant

### DISCUSSION THREAD

**Discussion by brux88** [JonPSmith/AuthPermissions.AspNetCore#27] [Q&A]
hi, I wanted to congratulate myself for his blog and his articles are a source of inspiration. I wanted to create a modular app and set the multitenant management, both in a single db and in several db, by downloading the package from nuget it seems that this functionality is not there, is it possible?

    options.TenantType = TenantTypes.SingleLevel | TenantTypes.AddSharding;

**Comment by JonPSmith** [comment]
Hi @brux88,

I'm just working through the release of version 3 of this library which adds sharding, but I decided update the documentation  before I release the NuGet. That's because there is quite a lot of new documentation. Problem is I got covid this week and feel like I have bad flu (I'm OK as I have three jabs). I think version 3 should be out next week (I hope) and another article explaining how it works. 

Also there are some extra parts, like how to update the appsettings file on a running application and how to create a Azure database via code, which will be out later

**Reply by JonPSmith** [reply]
Version 3.0.0 is now out
